<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>669</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout"/>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="openFile"/>
    <addaction name="openDatabase"/>
    <addaction name="separator"/>
    <addaction name="saveFile"/>
    <addaction name="saveDatabase"/>
    <addaction name="separator"/>
    <addaction name="logout"/>
    <addaction name="separator"/>
    <addaction name="exit"/>
   </widget>
   <widget class="QMenu" name="menuTransform">
    <property name="title">
     <string>Transform</string>
    </property>
    <widget class="QMenu" name="menuHistogram_equalization">
     <property name="title">
      <string>Histogram equalization</string>
     </property>
     <addaction name="actionHistogramLinear"/>
     <addaction name="actionHistogramAdaptive"/>
    </widget>
    <widget class="QMenu" name="menuLaplacian">
     <property name="title">
      <string>Laplacian</string>
     </property>
     <addaction name="actionLaplacian"/>
     <addaction name="actionDiagonalLaplacian"/>
     <addaction name="actionHLaplacian"/>
     <addaction name="actionVLaplacian"/>
     <addaction name="actionBLaplacian"/>
    </widget>
    <widget class="QMenu" name="menuPrewitt">
     <property name="title">
      <string>Prewitt</string>
     </property>
     <addaction name="actionHPrewitt"/>
     <addaction name="actionVPrewitt"/>
     <addaction name="actionBPrewitt"/>
    </widget>
    <widget class="QMenu" name="menuSobel">
     <property name="title">
      <string>Sobel</string>
     </property>
     <addaction name="actionHSobel"/>
     <addaction name="actionVSobel"/>
     <addaction name="actionBSobel"/>
    </widget>
    <widget class="QMenu" name="menuKirsch">
     <property name="title">
      <string>Kirsch</string>
     </property>
     <addaction name="actionHKirsch"/>
     <addaction name="actionVKirsch"/>
     <addaction name="actionBKirsch"/>
    </widget>
    <widget class="QMenu" name="menuSegmentations">
     <property name="title">
      <string>Segmentations</string>
     </property>
     <addaction name="actionOtsuThreshold"/>
     <addaction name="actionWatershed"/>
    </widget>
    <widget class="QMenu" name="menuMorphology">
     <property name="title">
      <string>Morphology</string>
     </property>
     <addaction name="actionErosion"/>
     <addaction name="actionDilation"/>
     <addaction name="actionClosing"/>
     <addaction name="actionOpening"/>
    </widget>
    <addaction name="actionGrayScale"/>
    <addaction name="actionLowPass"/>
    <addaction name="actionHighPass"/>
    <addaction name="menuHistogram_equalization"/>
    <addaction name="menuLaplacian"/>
    <addaction name="menuPrewitt"/>
    <addaction name="menuSobel"/>
    <addaction name="menuKirsch"/>
    <addaction name="menuSegmentations"/>
    <addaction name="menuMorphology"/>
    <addaction name="actionPseudocoloration"/>
    <addaction name="actionColorInversion"/>
   </widget>
   <widget class="QMenu" name="menuHistory">
    <property name="title">
     <string>History</string>
    </property>
    <addaction name="undo"/>
    <addaction name="redo"/>
    <addaction name="separator"/>
    <addaction name="clearHistory"/>
    <addaction name="original"/>
   </widget>
   <widget class="QMenu" name="menuQuick_transforms">
    <property name="title">
     <string>Quick transforms</string>
    </property>
    <addaction name="actionQuick1"/>
    <addaction name="actionQuick2"/>
    <addaction name="actionQuick3"/>
    <addaction name="actionQuick4"/>
    <addaction name="actionQuick5"/>
    <addaction name="actionQuick6"/>
   </widget>
   <widget class="QMenu" name="menuZoom">
    <property name="title">
     <string>Zoom</string>
    </property>
    <addaction name="increaseZoom"/>
    <addaction name="decreaseZoom"/>
    <addaction name="resetZoom"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHistory"/>
   <addaction name="menuZoom"/>
   <addaction name="menuTransform"/>
   <addaction name="menuQuick_transforms"/>
  </widget>
  <action name="openFile">
   <property name="text">
    <string>Import from file...</string>
   </property>
  </action>
  <action name="openDatabase">
   <property name="text">
    <string>Import from database...</string>
   </property>
  </action>
  <action name="saveFile">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save to file...</string>
   </property>
  </action>
  <action name="saveDatabase">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save to database...</string>
   </property>
  </action>
  <action name="logout">
   <property name="text">
    <string>Logout</string>
   </property>
  </action>
  <action name="exit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionGrayScale">
   <property name="text">
    <string>Grayscale</string>
   </property>
   <property name="iconText">
    <string>GrayScale</string>
   </property>
  </action>
  <action name="actionana">
   <property name="text">
    <string>ana</string>
   </property>
  </action>
  <action name="actionLowPass">
   <property name="text">
    <string>Low pass (Blurring)</string>
   </property>
  </action>
  <action name="actionHighPass">
   <property name="text">
    <string>High pass (Edge detection)</string>
   </property>
  </action>
  <action name="actionHistogramLinear">
   <property name="text">
    <string>Linear</string>
   </property>
  </action>
  <action name="actionHistogramAdaptive">
   <property name="text">
    <string>Adaptive</string>
   </property>
  </action>
  <action name="actionLaplacian">
   <property name="text">
    <string>Laplacian</string>
   </property>
  </action>
  <action name="actionHLaplacian">
   <property name="text">
    <string>Horizontal Laplacian</string>
   </property>
  </action>
  <action name="actionVLaplacian">
   <property name="text">
    <string>Vertical Laplacian</string>
   </property>
  </action>
  <action name="actionHPrewitt">
   <property name="text">
    <string>Horizontal Prewitt</string>
   </property>
  </action>
  <action name="actionVPrewitt">
   <property name="text">
    <string>Vertical Prewitt</string>
   </property>
  </action>
  <action name="actionHSobel">
   <property name="text">
    <string>Horizontal Sobel</string>
   </property>
  </action>
  <action name="actionVSobel">
   <property name="text">
    <string>Vertical Sobel</string>
   </property>
  </action>
  <action name="actionHKirsch">
   <property name="text">
    <string>Horizontal Kirsch</string>
   </property>
  </action>
  <action name="actionVKirsch">
   <property name="text">
    <string>Vertical Kirsch</string>
   </property>
  </action>
  <action name="undo">
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="redo">
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
  </action>
  <action name="clearHistory">
   <property name="text">
    <string>Clear history</string>
   </property>
  </action>
  <action name="actionDiagonalLaplacian">
   <property name="text">
    <string>Diagonal Laplacian</string>
   </property>
  </action>
  <action name="actionBLaplacian">
   <property name="text">
    <string>Biderectional Laplacian</string>
   </property>
  </action>
  <action name="actionBPrewitt">
   <property name="text">
    <string>Biderectional Prewitt</string>
   </property>
  </action>
  <action name="actionBSobel">
   <property name="text">
    <string>Biderectional Sobel</string>
   </property>
  </action>
  <action name="actionBKirsch">
   <property name="text">
    <string>Biderectional Kirsch</string>
   </property>
  </action>
  <action name="actionOtsuThreshold">
   <property name="text">
    <string>Otsu threshold</string>
   </property>
  </action>
  <action name="actionPseudocoloration">
   <property name="text">
    <string>Pseudocoloration</string>
   </property>
  </action>
  <action name="original">
   <property name="text">
    <string>Original image</string>
   </property>
  </action>
  <action name="actionErosion">
   <property name="text">
    <string>Erosion</string>
   </property>
  </action>
  <action name="actionDilation">
   <property name="text">
    <string>Dilation</string>
   </property>
  </action>
  <action name="actionClosing">
   <property name="text">
    <string>Closing</string>
   </property>
  </action>
  <action name="actionOpening">
   <property name="text">
    <string>Opening</string>
   </property>
  </action>
  <action name="actionWatershed">
   <property name="text">
    <string>Watershed</string>
   </property>
  </action>
  <action name="actionQuick5">
   <property name="text">
    <string>Cluster detection</string>
   </property>
  </action>
  <action name="actionQuick1">
   <property name="text">
    <string>Edge detection</string>
   </property>
  </action>
  <action name="actionQuick2">
   <property name="text">
    <string>Kirsch amplification</string>
   </property>
  </action>
  <action name="actionQuick4">
   <property name="text">
    <string>Heatmap apmplification</string>
   </property>
  </action>
  <action name="actionQuick3">
   <property name="text">
    <string>Edge binarization</string>
   </property>
  </action>
  <action name="actionQuick6">
   <property name="text">
    <string>Inverted Binarization</string>
   </property>
  </action>
  <action name="increaseZoom">
   <property name="text">
    <string>Increase</string>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
  </action>
  <action name="decreaseZoom">
   <property name="text">
    <string>Decrease</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="resetZoom">
   <property name="text">
    <string>Reset</string>
   </property>
  </action>
  <action name="actionColorInversion">
   <property name="text">
    <string>Color Inversion</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
